### PR TITLE
Increase zypper timeout in publiccloud migration

### PR DESCRIPTION
**Pull request by yarunachalam**
Due to this issue https://openqa.suse.de/tests/9802122#step/migration/168 
Increase zypper timeout in publiccloud migration
Test Run: with CASEDIR=https://github.com/yarunachalam/os-autoinst-distri-opensuse.git#timeout https://openqa.suse.de/tests/9802495
https://openqa.suse.de/tests/9802500
https://openqa.suse.de/tests/9802497

### CODE REVIEW

**Review by Avinesh — APPROVED** [review]

**Review by okurz — CHANGES_REQUESTED** [review]
What you put into the description of the pull request could also be in the commit message details. I don't know how you created your pull request but your git commit messages has *only* the subject line while the pull request description has more details. Please keep in mind that the github pull request description is only visible on github, the commit can be considered permanent information storage.

I can recommend the tool [hub](https://github.com/github/hub) (zypper in rubygem-hub) for easier PR creation. Also, I myself use a script [git-pr-last](http://github.com/okurz/scripts/tree/master/git-pr-last) to create a PR with proper description for these simple one-commit PRs:
```bash
$ cat $(which git-pr-last )
#!/bin/sh -e
target="${target:-"$USER"}"
git push $target && git show --no-patch --format=%B | hub pull-request -F -
```

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

**Comment by yarunachalam** [comment]
updated the commit description also

File: tests/publiccloud/migration.pm [hunk 1]
```diff
@@ -36,7 +36,8 @@ sub run {
 
     record_info('installs SLE15-Migration and suse-migration-sle15-activation');
 
-    $instance->run_ssh_command(cmd => "sudo zypper -n in SLES15-Migration suse-migration-sle15-activation", timeout => 300);
+    $instance->run_ssh_command(cmd => "sudo zypper -n in SLES15-Migration suse-migration-sle15-activation", timeout => 360);
```
Comment (grisu48) [inline tests/publiccloud/migration.pm:39]: I fixed this already yesterday in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15805. Are those additional 60 seconds really necessary?

**Comment by grisu48** [comment]
Why are there two commits?

File: tests/publiccloud/migration.pm (hunk 1 above)
Comment (yarunachalam) [reply tests/publiccloud/migration.pm:39]: Earlier it was 90 , I had issues with Azure image. It used to timeout https://openqa.suse.de/tests/9802122#step/migration/168

Let me test for couple of days. If 300 works, I can leave it as it's

Comment (yarunachalam) [reply tests/publiccloud/migration.pm:39]: Also I would like to understand, What was your intention to increase the timeout in migration script ??

**Comment by yarunachalam** [comment]
> Why are there two commits?

squashed the commit

**Comment by okurz** [comment]
please phrase the git commit message in imperative mood, e.g. "Increase zypper timeout in publiccloud migration"

File: tests/publiccloud/migration.pm (hunk 1 above)
Comment (grisu48) [reply tests/publiccloud/migration.pm:39]: I'm on QA-C job review duty this week, so yes, I'm monitoring all jobs that are in our production pipeline. I saw the migration tests failing, filed https://progress.opensuse.org/issues/119602 for it and provided a quickfix in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15805.

**Comment by yarunachalam** [comment]
> please phrase the git commit message in imperative mood, e.g. "Increase zypper timeout in publiccloud migration"

Done

**Comment by yarunachalam** [comment]
> Please rebase to exclude the merge commit. And it looks like you lost the git commit message subject line. I suggest you look into the git history to understand how other commit messages look like as reference

**Comment by yarunachalam** [comment]
Done

**Review by okurz — APPROVED** [review]

**Comment by okurz** [comment]
I already approved and you asked for review again but there are no changes. Maybe you forgot to push changes after conflict resolution?

**Comment by yarunachalam** [comment]
already done with other PR. Can be closed

**Pull request closed without merging**